### PR TITLE
Feature/e2e workbench pipeline

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        const valueInEditor = '100';
+        await settingsPage.changeCommandsInPipeline(pipelineValues[2]);
+        // Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await workbenchPage.sendCommandInWorkbench(commandForSend);
+        await t.typeText(workbenchPage.queryInput, commandForSend, { replace: true, paste: true });
+        await t.pressKey('enter');
+        // 'Verify that user can interact with the Editor
+        await t.expect(workbenchPage.queryInputScriptArea.textContent).contains(valueInEditor, {timeout: 5000});
+    });
```
Comment (AlenaNaboka) [inline tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts:80]: Can you please add one more test with validation of order results after run commands in pipelines?

File: tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    .meta({ rte: rte.standalone })('Verify that only chosen in pipeline number of commands is loading at the same time in Workbench', async t => {
+        await settingsPage.changeCommandsInPipeline(pipelineValues[2]);
+        // Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await workbenchPage.sendCommandInWorkbench(commandForSend, 0.01);
+        // Verify that only selected pipeline number of commands are loaded at the same time
+        await t.expect(workbenchPage.loadedCommand.count).eql(Number(pipelineValues[2]), 'The number of sending commands is incorrect');
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can see spinner over the disabled and shrunk Run button and grey preloader for each command in Workbench History', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts:58]: Test name is too long

**Review by ofersteinberg — COMMENTED** [review]
@vlad-dargel @AlenaNaboka, I could see you use with very long test name. A name, should be short and descriptive.
Examples: 
`Verify one can set only valid values in "Commands in pipeline"`
`Verify one can choose a few commands during loading`
ETC.

File: tests/e2e/pageObjects/workbench-page.ts [hunk 3]
```diff
@@ -93,6 +93,8 @@ export class WorkbenchPage {
     commandExecutionResultFailed = Selector('[data-testid=cli-output-response-fail]');
     chartViewTypeOptionSelected = Selector('[data-testid=view-type-selected-Plugin-redistimeseries__redistimeseries-chart]');
     runButtonToolTip = Selector('[data-testid=run-query-tooltip]');
+    loadedCommand = Selector('[class=euiLoadingContent__singleLine]');
+    runButtonSpinner = Selector('[class^=euiLoadingSpinner]');
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/pageObjects/workbench-page.ts]: thanks, added

File: tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts:80]: added 'Verify that command results are added to history in order most recent - on top' test

File: tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts]: fixed, thanks

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**